### ISSUE TITLE
Add Purpur support 

### ISSUE THREAD

**Issue by LichLord91** [itzg/docker-minecraft-server#746]
I believe their download page is similar to papermc , https://purpur.pl3x.net/downloads/#1.16.5

**Comment by itzg** [comment]
### Notes

Builds listing API is not published, but fairly intuitive. From network inspector, to get list of builds for a vanilla version:

```
GET https://purpur.pl3x.net/api/v1/purpur/1.16.5
```

with response JSON:
```json
{
"project":"purpur","version":"1.16.5",
"builds":{"latest":"967","all":["967","966","965","964","963","962","961","960","959","958","957"]}
}
```

Download API URL formed like `https://purpur.pl3x.net/api/v1/purpur/1.16.5/967/download`

**Comment by itzg** [comment]
Latest image now includes [Purpur support](https://github.com/itzg/docker-minecraft-server#running-a-purpur-server): 

```
> docker run -it --rm -e EULA=TRUE -e TYPE=PURPUR itzg/minecraft-server
[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxrwx    2 1000     1000          4096 Feb  7 17:28 /data'
[init] Resolved version given LATEST into 1.16.5
[init] Resolving type given PURPUR
[init] Downloading Purpur from https://purpur.pl3x.net/api/v1/purpur/1.16.5/969/download ...
```

**Comment by LichLord91** [comment]
Ill be testing this tonight ITZG. Thanks so much!

**Comment by LichLord91** [comment]
Realized this was only on latest and not multiarch-latest haha. I forked the repo merged the commits from master to multiarch-latest , updated the Dockerfile to build a arm64 build and tested it out on my raspberry pi 4. Can confirm working thusfar, though only on arm64 since i didn't test on any other arch type. ;)  I submitted a pull request to update the invalid server type as i ran into that issue first since i didn't read that it was only updated for latest docker tag atm. Saw that you were missing Purpur and Tuinity from the invalid server type error log and so i made a small contribution as thanks for the hard work 🙏
